robottelo/ui/discoveryrules.py

```diff
@@ -65,8 +52,11 @@ def search(self, name):
         """Searches existing discovery rule from UI. It is necessary to use
         custom search as we don't have both search bar and search button there.
         """
+        # Remove entire method once BZ 1233135 is fixed
         self.navigate_to_entity()
         strategy, value = self._search_locator()
+        if len(name) > 32:
```

Comment:
Logic has been changed for that entity and we still don't have search box for it according to 1233135 BZ. It doesn't make sense to change base class for that specific case